doc/Analytics.xml

```diff
@@ -499,6 +501,480 @@
+        &lt;tt:Scale x="0.003125" y="0.00416667"/&gt;
+      &lt;/tt:Transformation&gt;
+      &lt;tt:Object ObjectId="15"&gt;
+        &lt;tt:Appearance&gt;
+          &lt;tt:Shape&gt;
+            &lt;tt:BoundingBox left="20.0" top="80.0" right="100.0" bottom="30.0"/&gt;
+            &lt;tt:CenterOfGravity x="60.0" y="50.0"/&gt;
+            &lt;tt:Polygon&gt;
+              &lt;tt:Point x="20.0" y="30.0"/&gt;
+              &lt;tt:Point x="100.0" y="30.0"/&gt;
+              &lt;tt:Point x="100.0" y="80.0"/&gt;
+              &lt;tt:Point x="20.0" y="80.0"/&gt;
+            &lt;/tt:Polygon&gt;
+          &lt;/tt:Shape&gt;
+          &lt;tt:Color&gt;
+            &lt;tt:ColorCluster&gt;
+              &lt;tt:Color X="213" Y="135" Z="126" /&gt;
+              &lt;tt:Weight&gt;0.9&lt;/tt:Weight&gt;
+              &lt;acme:ColorName&gt;White&lt;/acme:ColorName&gt;
+            &lt;/tt:ColorCluster&gt;
+            &lt;tt:ColorCluster&gt;
+              &lt;tt:Color X="102" Y="176" Z="119" /&gt;
+              &lt;tt:Weight&gt;0.5&lt;/tt:Weight&gt;
+              &lt;acme:ColorName&gt;Blue&lt;/acme:ColorName&gt;
+            &lt;/tt:ColorCluster&gt;
+          &lt;/tt:Color&gt;
+          &lt;tt:Class&gt;
+            &lt;tt:Type Likelihood="0.8"&gt;Human&lt;/tt:Type&gt;
+          &lt;/tt:Class&gt;
+          &lt;tt:HumanFace&gt;
+            &lt;fc:Gender&gt;Male&lt;/fc:Gender&gt;
+            &lt;fc:Age&gt;
+              &lt;tt:Min&gt;20&lt;/tt:Min&gt;
+              &lt;tt:Max&gt;30&lt;/tt:Max&gt;
+            &lt;/fc:Age&gt;
+            &lt;fc:Accessory&gt;
+              &lt;fc:Opticals&gt;
+                &lt;fc:Wear&gt;True&lt;/fc:Wear&gt;
+              &lt;/fc:Opticals&gt;
+              &lt;fc:Hat&gt;
+                &lt;fc:Wear&gt;True&lt;/fc:Wear&gt;
+              &lt;/fc:Hat&gt;
+            &lt;/fc:Accessory&gt;
+          &lt;/tt:HumanFace&gt;
+          &lt;tt:HumanBody&gt;
+            &lt;bd:Clothing&gt;
+              &lt;bd:Tops&gt;
+                &lt;bd:Category&gt;LongSleeve&lt;/bd:Category&gt;
+              &lt;/bd:Tops&gt;
+              &lt;bd:Bottoms&gt;
+                &lt;bd:Category&gt;Shorts&lt;/bd:Category&gt;
+              &lt;/bd:Bottoms&gt;
+            &lt;/bd:Clothing&gt;
+            &lt;bd:Belonging&gt;
+              &lt;bd:Bag&gt;
+                &lt;bd:Category&gt;Backpack&lt;/bd:Category&gt;
+              &lt;/bd:Bag&gt;
+            &lt;/bd:Belonging&gt;
+          &lt;/tt:HumanBody&gt;
+        &lt;/tt:Appearance&gt;
+      &lt;/tt:Object&gt;
+    &lt;/tt:Frame&gt;
+  &lt;/tt:VideoAnalytics&gt;
+&lt;/tt:MetadataStream&gt;</programlisting>
+
+        <para>This data will be published under this topic:</para>
+
+        <programlisting>MyDevice/onvif-mj/VideoAnalytics/1/MyClassifier</programlisting>
+
+        <para>Where "MyDevice" is the given TopicPrefix and "1" is the
+        analytics configuration token. Shown below is how the sample metadata
```

Comment:
According to the table above topic should have profile token. 

**"1" is the  analytics configuration token   ---> "1" is the profile token**